codeworld-server/src/Main.hs

```diff
@@ -345,39 +324,13 @@ moveProjectHandler = private $ \userId ctx -> do
                 toFile = projectDir </> moveToDir </> projectFile projectId
             liftIO $ ensureProjectDir mode userId moveToDir projectId
             liftIO $ copyFile file toFile
-            empty <-
-                liftIO $
-                fmap
-                    (\l1 ->
-                         length l1 == 3 &&
-                         sort l1 ==
-                         sort [".", "..", takeFileName $ projectFile projectId])
-                    (getDirectoryContents
-                         (dropFileName $ moveFromDir </> projectFile projectId))
-            liftIO $
-                if empty
-                    then removeDirectoryIfExists
-                             (dropFileName $
-                              moveFromDir </> projectFile projectId)
-                    else removeFileIfExists $
-                         moveFromDir </> projectFile projectId
+            liftIO $ removeFileIfExists $ moveFromDir </> projectFile projectId
```

Comment:
Isn't this just `liftIO $ removeFileIfExists file` ?

Reply:
yes, fixed.